### DISCUSSION TITLE
Which value was used for masked values of SpecAugment?

### DISCUSSION THREAD

**Discussion by jumon** [openai/whisper#730] [Q&A] [answered]
Thank you for the great work and for making it open-source!

I am currently trying to fine-tune the Whisper large-v2 model with SpecAugment and I am wondering which value was used for masked values during training. Was it 0? My concern with using 0 for masking is that it is also used for padding audio, and this could potentially cause the model to "hallucinate" at the end of shorter audio files (less than 30 seconds) during recognition, since the model was trained to predict masked parts that may have overlapped with padding values.

**Comment by jongwook** [comment, marked as answer]
Apologies for missing this post -- this is a totally valid point and I should fix the zero-padding method in `transcribe()` (also answered in #838)

**Reply by jumon** [reply]
Thanks!
